test/test-applications/regression/StackExchange.Redis.StackOverflowException/StackExchange.Redis.StackOverflowException.csproj

```diff
@@ -3,6 +3,8 @@
   <PropertyGroup>
     <TargetFrameworks>net461;netcoreapp2.1;netcoreapp3.0;netcoreapp3.1;net5.0</TargetFrameworks>
     <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
+    <!-- Ignore  Found conflicts between different versions of "System.Diagnostics.PerformanceCounter" that could not be resolved. -->
```

Comment:
Should we try to fix these conflicts instead of ignoring the warnings?

Reply:
Fixed in https://github.com/DataDog/dd-trace-dotnet/pull/1524/commits/c2a1e0d0120ac8915d6ab42f40fd590cf9c6f42d